src/transformers/models/bert_japanese/tokenization_bert_japanese.py

```diff
@@ -586,9 +589,12 @@ def __init__(
         else:
             raise ValueError("Invalid sudachi_split_mode is specified.")
 
-        self.sudachi = dictionary.Dictionary(
+        self.projection = sudachi_projection
+
+        sudachi_dictionary = dictionary.Dictionary(
             config_path=sudachi_config_path, resource_dir=sudachi_resource_dir, dict=sudachi_dict_type
-        ).create(self.split_mode)
+        )
+        self.sudachi = sudachi_dictionary.create(self.split_mode, projection=self.projection)
```

Comment:
as you mentioned this is not backward compatible as this was introduced later on in sudachi!

Reply:
I would use `if is_sudachi_projection_available` here, and else use the backward compatible beahviour!